setup/install.sh

```diff
@@ -126,8 +126,8 @@ fi
 # status for check requirements
 doil_status_okay
 
-doil_status_send_message_nl "Set host variable"
-doil_set_host
+doil_status_send_message "Adding group 'doil'"
+doil_system_add_group
 CHECK=$?
```

Comment:
Couldn't you just use `if [[ $? -ne 0 ]]` here (and in the other checks)?

Reply:
done!